tests/test_db/test_db.py

```diff
@@ -45,7 +45,8 @@ def test_items_equal_db(filename, total, entries):
 
         prop_json = cur.fetchall()
         prop_json = [item["_props"] for item in prop_json]
-        prop_json = sorted(prop_json, key=itemgetter("submitter_id"))
+        kwy_field = "submitter_id" if filename != "project" else "code"
```

Comment:
Would it be possible for you to update the typo in the variable name? I mean it is a small change, I'll leave it to you.